components/authz-service/server/v2/system.go

```diff
@@ -185,7 +185,6 @@ func SystemPolicies() []*storage.Policy {
 				},
 				Projects: []string{constants.AllProjectsID},
 			},
-			// TODO can't delete admin team from admin policy
```

Comment:
By removing this comment you are claiming that it is now true that you cannot delete the admin team, but is that true now...?

Reply:
We can't manage deletion of specific members of a policy with a policy (only general membership).  We'd need to modify the API. We could make a card for that if we think it's worth it.